CHANGES.rst

```diff
@@ -56,6 +56,8 @@ Specviz2d
 Bug Fixes
 ---------
 
+- Fixed histogram viewer breaking when its "Home" button is pressed within Plot Options. [#2529]
```

Comment:
So technically this isn't fixed by this PR but by glue-core v1.14.1, but maybe still worth leaving this here?

Reply:
Since the bug itself never saw a release, maybe we should just lump this PR in with one of the other changelog entries?  Maybe alongside #2498?

Reply:
If the bug never saw a release we could just remove any mention of it altogether from the changelog?